manual/cloud/README.md

```diff
@@ -110,7 +110,63 @@ database.
           }
           ```
 
-    c. Save and close the ConnectDatabase.java file.
+    c. **File-based configuration**. An alternative to the programmatic configuration method 
+    detailed above is to include the information required to connect in the driver's configuration 
+    file (`application.conf`). Merge the following options with any other options that you might 
+    want to include in the configuration file:
+    
+    ```hocon
+   basic {
+     # change this to match the target keyspace
+     session-keyspace = keyspace_name
```

Comment:
does it need to be specified? I think it will be taken from `secure-connect-bundle`

Reply:
We don't parse the keyspace anymore. We still parse the credentials though, even if they are now always absent.

Reply:
Ok, but even in that case, the session-keyspace is not required, right?
The client can specify keyspace later. I am not sure if we should mention that in the docs

Reply:
I am not sure about the keyspace actually. It was said at some point that users would have access to just one keyspace. Not sure if this is valid still. But it doesn't hurt to specify a keyspace anyways, so I would leave it that way. @emerkle826 do you know by chance if a keyspace is required for cloud?

Reply:
The last I was aware, you don't need to specify a keyspace, but I believe you are correct in that they will only have access to one keyspace (for now at least). From Jorge:
>I think there was consensus for not parsing the keyspace: users might be able to create their own keyspace in future versions of cloud and it will not be provided when switching to token-based auth

Reply:
In this case let's leave the keyspace example, it seems that at least for the initial Cloud version it will be necessary to set it since users will only have access to that specific keyspace.